### ISSUE TITLE
Accessibility audit: Missing Privacy notice and link

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#314]
## Describe the issue
We are asking for permission to store data in cookies without a privacy statement to inform users of what we'll do with it or their rights.

## Fix recommendations
- [x] Publish a privacy statement and link in the footer (@martinwake is there a generic one which just covers GA use?)

**Comment by martinwake** [comment]
The likely best option is a footer link that reads "Privacy" and links directly to https://www.gov.uk/government/organisations/department-for-work-pensions/about/personal-information-charter.

Internal guidance backs this up: https://intranet.dwp.gov.uk/policy/cookies-and-dwp-digital-services

Although we're not collecting much information and have no interest in our users' identities, depending on which cookies we set they _could_ be considered personal information: for example, I don't know if we collect this data but it's conceivable that things like IP address + user-agent data could be used in combination to identify someone.

**Comment by martinwake** [comment]
https://gitlab.com/dwp/dwp-design-system/main-site/-/merge_requests/179
